README.md

```diff
@@ -27,13 +35,13 @@ This project is still in development. The goal is to create a simple music playi
 5. Follow the beginning of [this guide](https://www.freecodecamp.org/news/create-a-discord-bot-with-python#how-to-create-a-discord-bot-account) to create a discord bot and generate an API token for it. 
 6. Create secrets.py using the example:
 		
-		$ cp secrets.py.example secrets.py
-	Then replace the empty `TOKEN` string with your generated token from the previous step.
+		$ cp .secrets.default .secrets
```

Comment:
I feel like we should use `.env.example` and `.env`. Is there a specific reason to use these names?

Reply:
We also seem to be missing the new environment file

Reply:
Yes, `*.example` is probably better. Regarding the `*`, any name goes; I'm thinking mostly that it's a config file, not an environment variable file. Additionally, the file should probably not be "hidden".